benchmark/results-2017-08-12/summary-2017-08-12.txt

```diff
@@ -0,0 +1,125 @@
+bq: 1.1.0
+bq-0: 0.3.0
+bull: 3.0.0-rc.5
+kue: 0.11.6
+
+across all benchmarks =>
+  [2550 :: 13511 {median: 4676, mean: 5470.6}]
+
+by node version
+ 6.9.1 => [2804 :: 13147 {median: 4697, mean: 5312.2}]
+6.11.2 => [2690 :: 13511 {median: 4676, mean: 5288.5}]
+ 7.6.0 => [2550 :: 12680 {median: 4496, mean: 5077.9}]
+7.10.1 => [2568 :: 12837 {median: 4500, mean: 5125.7}]
+ 8.2.1 => [3854 :: 12999 {median: 5308, mean: 5946.9}]
+ 8.3.0 => [4046 :: 12548 {median: 5275, mean: 6072.6}]
+
+by library
+    bq => [2712 :: 6568 {median: 3968, mean: 3928.2}]
+  bq-0 => [2695 :: 6516 {median: 3939, mean: 3928.0}]
+bq-min => [2550 :: 6111 {median: 3854, mean: 3690.1}]
+  bull => [4676 :: 7711 {median: 5567, mean: 5839.2}]
+   kue => [8283 :: 13511 {median: 9285, mean: 9967.7}]
+
+by concurrency
+ 1 => [4136 :: 13511 {median: 6126, mean: 7107.8}]
+ 5 => [2894 :: 10159 {median: 4580, mean: 5339.3}]
+20 => [2626 :: 9407 {median: 4190, mean: 4786.9}]
+50 => [2550 :: 9194 {median: 4046, mean: 4648.6}]
+
+split by node version, then library
+ 6.9.1 =>
+        bq => [2916 :: 4992 {median: 3485, mean: 3634.9}]
+      bq-0 => [2927 :: 5034 {median: 3536, mean: 3643.5}]
+    bq-min => [2804 :: 4621 {median: 3220, mean: 3396.5}]
+      bull => [4697 :: 7418 {median: 5567, mean: 5657.9}]
+       kue => [8751 :: 13147 {median: 9911, mean: 10228.3}]
+6.11.2 =>
+        bq => [2877 :: 4916 {median: 3441, mean: 3549.7}]
+      bq-0 => [2861 :: 4927 {median: 3432, mean: 3557.1}]
+    bq-min => [2690 :: 4553 {median: 3016, mean: 3286.4}]
+      bull => [4676 :: 7408 {median: 5584, mean: 5673.7}]
+       kue => [8673 :: 13511 {median: 9907, mean: 10375.7}]
+ 7.6.0 =>
+        bq => [2712 :: 4593 {median: 3210, mean: 3345.0}]
+      bq-0 => [2714 :: 4603 {median: 3234, mean: 3354.4}]
+    bq-min => [2550 :: 4273 {median: 2894, mean: 3096.5}]
+      bull => [4954 :: 7446 {median: 5827, mean: 5856.7}]
+       kue => [8338 :: 12680 {median: 9324, mean: 9736.9}]
+7.10.1 =>
```

Comment:
If I'm interpreting these numbers correctly, it looks like bee-queue is significantly faster on Node 7 and even 6 than on 8, while others are more consistent across Node versions. Any idea why that might be? Is it worth pointing out in the readme?